### PR TITLE
File names fixed. Hashing of vector y fixed.

### PR DESCRIPTION
**Pull request by miha-stopar**
Signed-off-by: Miha Stopar <miha.stopar@xlab.si>

### CODE REVIEW

File: innerprod/fullysec/dmcfe.go [hunk 1]
```diff
@@ -74,13 +74,7 @@ func (c *DMCFEClient) Encrypt(x *big.Int, label string) (*bn256.G1, error) {
...
 	var yRepr []byte
 	for i := 0; i < len(y); i++ {
-		yRepr = append(yRepr, y[i].Bytes()...)
-		yiAbs := new(big.Int).Abs(y[i])
-		if yiAbs.Cmp(y[i]) == 0 {
-			yRepr = append(yRepr, 1)
-		} else {
-			yRepr = append(yRepr, 2)
-		}
+		yRepr = append(yRepr, []byte(y[i].String())...) // don't use directly Bytes() because it is the same for a negative counterpart
```
Comment (janhartman) [inline innerprod/fullysec/dmcfe.go]: How come you changed it to this? The change you are proposing is less efficient (i.e. we are hashing a decimal string representation of the big integer) - this means using more memory, although if it doesn't incur a performance penalty, it should be OK. I agree that it is cleaner than what we had before though.

Comment (miha-stopar) [reply innerprod/fullysec/dmcfe.go]: In previous version there would be 1 or 2 at the beginning of each vector coordinate (kind of a pattern) - it shouldn't cause problems once it is hashed but you never know. I think it is safer as it is now.

Comment (janhartman) [reply innerprod/fullysec/dmcfe.go]: Could it be a problem if we don't separate numbers? E.g. : in one case, y would be `[12, 345]` and in another, `[123, 45]`. These give the same `yRepr` and therefore the same hash.

**Comment by janhartman** [comment]
At least we got two spelling fixes :)

**Pull request merged**